extern/sector-storage/sched.go

```diff
@@ -393,7 +393,8 @@ func (sh *scheduler) trySched() {
 				}
 
 				// TODO: allow bigger windows
-				if !windows[wnd].allocated.canHandleRequest(needRes, windowRequest.worker, "schedAcceptable", worker.info.Resources) {
+				ignoringResources := worker.info.IgnoreResources
+				if !ignoringResources && !windows[wnd].allocated.canHandleRequest(needRes, windowRequest.worker, "schedAcceptable", worker.info.Resources) {
```

Comment:
Would pass full `worker.info` into `canHandleRequest` and handle this condition there